E2E/DURABLE-FUNC-ORDER-PROCESSING/OrderProcessor/host.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "excludedTypes": "Request"
+      },
+      "enableLiveMetricsFilters": true,
+      "logLevel": {
+        "Host.Triggers.DurableTask": "Information"
+      }
+    }
+  },
+  "extensions": {
+    "durableTask": {
+      "storageProvider": {
+        "type": "AzureStorage"
```

Comment:
The recommendation is to also lower the default maxQueuePollingInterval right? We should do that here.